public-ui/src/app/data-browser/chart/chart.component.ts

```diff
@@ -528,5 +530,14 @@ export class ChartComponent implements OnChanges {
     };
 
   }
+  public getAgeChartTitle(component: string) {
+    if (component === 'ehr') {
```

Comment:
nit: seems like a good use case for an enum

Reply:
Changed.